fixtures/glp/knv-arc.json

```
@@ -0,0 +1,1730 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "KNV Arc",
+  "categories": ["Effect", "Color Changer", "Strobe"],
+  "meta": {
+    "authors": ["Flo Edelmann"],
+    "createDate": "2019-04-05",
+    "lastModifyDate": "2019-04-05"
+  },
+  "links": {
+    "manual": [
+      "https://www.glp.de/en/products?format=raw&task=download&fid=385"
+    ],
+    "productPage": [
+      "https://www.glp.de/en/products/creative-led-system/knv-cube"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=JavZqUiT_ik",
+      "https://www.youtube.com/watch?v=ZlpA7hrJNSw"
+    ]
+  },
+  "physical": {
+    "dimensions": [250, 250, 183],
```

```suggestion
    "dimensions": [250, 250, 180],
```

Specific for the Arc variant.